modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/ColorPicker/ColorPicker.es.js

```diff
@@ -86,23 +86,76 @@ const ColorPicker = ({
 	spritemap,
 	value,
 	...otherProps
-}) => (
-	<FieldBase
-		name={name}
-		readOnly={readOnly}
-		spritemap={spritemap}
-		{...otherProps}
-	>
-		<ClayColorPickerWithState
-			inputValue={value ? value : predefinedValue}
+}) => {
+	let _event;
+
```

Comment:
@marcosapmf, can we rename it to `colorDropDownClickEvent` instead?

In addition, can we remove the blank line on line 91?

Reply:
Thanks for reviewing, @natocesarrego. `colorDropDownClickEvent` looks much better :smile: